hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,6 @@
+RELEASE_TYPE: patch
+
+Fixes bug in `hypothesis.extra.numpy.arrays` where the desired data type is not
```

Comment:
This does change some observable behavior, so I think it's worth a slightly longer description.  I've also got a few formatting changes, but instead of trying to describe Sphinx punctuation soup, I'll just put my suggested version below:

```rst
RELEASE_TYPE: minor

This release changes some inconsistent behavior of :func:`~hypothesis.extra.numpy.arrays` 
from the Numpy extra when asked for an array of ``shape=()``.
:func:`~hypothesis.extra.numpy.arrays` will now always return a Numpy
:class:`~numpy:numpy.ndarray`, and the array will always be of the requested dtype.

Thanks to Ryan Turner for this change.
```

Is this still fully correct?

Reply:
yes, should I include the thanks in the file too?

Reply:
Yes please - it's a little weird to write it yourself, but we do want it to appear in the changelog if you don't mind!